cocos/core/renderer/core/render-window.ts

```diff
@@ -226,6 +229,12 @@ export class RenderWindow {
         }
     }
 
+    /**
+     * @en Extract all render cameras attached to the render window to the output cameras list
+     * @zh 将所有挂载到当前渲染窗口的摄像机存储到输出列表参数中
+     * @param cameras @en The output cameras list, should be empty before invoke this function
+     *                @zh 输出相机列表参数，传入时应该为空
```

Comment:
Can english and Chinese comments in different lines?

Reply:
Yes